packages/sanes-chrome-extension/src/routes/paths.unit.spec.ts

```diff
@@ -1,35 +0,0 @@
-import { ACCOUNT_STATUS_ROUTE, initialUrl, LOGIN_ROUTE, REQUEST_ROUTE, WELCOME_ROUTE } from "./paths";
-
```

Comment:
Why this unrelated deletion of `packages/sanes-chrome-extension/src/routes/paths.unit.spec.ts`?

Reply:
oops!